cypress/support/step-definitions/split-button-steps.js

```diff
@@ -120,36 +121,19 @@ When('I hover mouse onto split button', () => {
 
 When('I click {string} element of Split Button component', (element) => {
   switch (element) {
-    case 'first':
+    case 'main':
```

Comment:
could we use if / else here ? If (element === 'main') ?

Reply:
fixed